### ISSUE TITLE
Ignored unknown kwarg option direction, while running run_mlm.py (pytorch)

### ISSUE THREAD

**Issue by saahiluppal** [huggingface/transformers#15358]
transformers_version == '4.16.0.dev0'

using script ```run_mlm.py``` (pytorch)
with following parameters
```
!python run_mlm.py \
    --output_dir "./test-roberta-base" \
    --model_type "roberta" \
    --config_name "./test-roberta-base" \
    --tokenizer_name "./test-roberta-base" \
    --train_file "./training_data/test.txt" \
    --max_seq_length 128 \
    --line_by_line \
    --do_train \
    --weight_decay 0.01 \
    --per_device_train_batch_size 256 \
    --per_device_eval_batch_size 256 \
    --learning_rate 3e-4 \
    --warmup_steps 1000 \
    --overwrite_output_dir \
    --num_train_epochs 18 \
    --adam_beta1 0.9 \
    --adam_beta2 0.98 \
    --logging_steps 500 \
    --save_steps 2500 \
    --eval_steps 2500
```

when i pass in the parameter ```--line_by_line```, this happens
```
Running tokenizer on dataset line_by_line:   0%| | 66/1536000 [00:01<6:13:50, 68Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Running tokenizer on dataset line_by_line:   0%| | 76/1536000 [00:01<5:35:58, 76Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Running tokenizer on dataset line_by_line:   0%| | 85/1536000 [00:01<6:44:46, 63Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Ignored unknown kwarg option direction
Running tokenizer on dataset line_by_line:   0%| | 95/1536000 [00:01<6:01:37, 70Ignored unknown kwarg option direction
```

my train_file is in the form
```
sentence1
sentence2
sentence3
and so on...
```

is this expected?

PS: 
using custom tokenizer made via
```
tokenizer = ByteLevelBPETokenizer(lowercase=True)

tokenizer.train(
    files=paths,
    vocab_size=50265,
    min_frequency=10,
    special_tokens=["<s>", "<pad>", "</s>", "<unk>", "<mask>"]
)

tokenizer.save("./test-roberta-base/tokenizer.json")
```

**Comment by LysandreJik** [comment]
cc @sgugger

**Comment by sgugger** [comment]
Looks like a problem in the tokenizers. What's your version of the Tokenizers library?

**Comment by saahiluppal** [comment]
@sgugger
tokenizers.__version__ = '0.9.4'

**Comment by sgugger** [comment]
I think you might need to upgrade it.

**Comment by saahiluppal** [comment]
Okay, upgraded tokenizers to 0.11.0.
i can confirm that running run_mlm.py script doesn't throw that error anymore.

Thanks.

**Comment by HelloRusk** [comment]
In my case 0.11.0 still threw error but 0.11.5 worked

**Comment by OpenWaygate** [comment]
Same problem
```
# Name                    Version                   Build  Channel
tokenizers                0.10.3           py38hb317417_1
```

**Comment by jayeshchoudhari** [comment]
Same issue even with tokenizers version 0.11.6

**Comment by stevetracvc** [comment]
Any other options here? Which transformers version will properly work with tokenizers 0.10.x? Using conda, you can't install tokenizers version 0.11.x with transformers 4.x. I tried forcing it, but got errors regarding openssl v3 when trying to import tokenizers.

```
conda search transformers --info -c conda-forge
...
transformers 4.16.2 pyhd8ed1ab_0
...
dependencies: 
  - tokenizers >=0.10.1,<0.11
```

and

```
conda search transformers --info -c huggingface
...
transformers 4.14.1 pyhd3eb1b0_0
...
dependencies: 
  - tokenizers >=0.10.1,<0.11
```

**Comment by shrishabh** [comment]
Downgrading to transformers 4.15.0 fixed the issue for me! Thanks!

**Comment by paffell** [comment]
Downgrading helped. Thank you

**Comment by aissak21** [comment]
> 

I get the same error and am using `conda install transformers==4.15.0` to downgrade but get `PackagesNotFoundError: The following packages are not available from current channels: - transformers==4.15.0`

**Comment by aissak21** [comment]
The command `conda install -c huggingface -c conda-forge transformers==4.15.0` worked for me.
